lms/static/js/instructor_dashboard/membership.js

```diff
@@ -966,11 +995,28 @@ such that the value can be defined later than this assignment (file load order).
                 authList = $opt.data('auth_list');
                 authList.$container.addClass('active');
                 authList.re_view();
+
+                // On Change update the Group Moderation list
+                if ($opt.text() === 'Discussion Group Community TA') {
```

Comment:
I am applying this.  I like the result of this and think it is more reliable.  Thank you.